cli/lib/exec/spawn.js

```diff
@@ -74,7 +74,15 @@ const isContainerVulkanDriverWarning = /^Warning: vkCreateInstance/
 
 const isContainerVulkanStack = /^\s*at (CheckVkSuccessImpl|CreateVkInstance|Initialize|Create|operator).+(VulkanError|BackendVk).cpp/
 
-const GARBAGE_WARNINGS = [isXlibOrLibudevRe, isHighSierraWarningRe, isRenderWorkerRe, isDbusWarning, isCertVerifyProcBuiltin, isMacOSElectronWebSwapCGLLayerWarning, isHostVulkanDriverWarning, isContainerVulkanDriverWarning, isContainerVulkanStack]
+/**
+ * In Electron 32.0.0 a new debug scenario log message started appearing when iframes navigate to about:blank. This is a benign message.
+ * https://github.com/electron/electron/issues/44368
+ * Sample:
+ * [78887:1023/114920.074882:ERROR:debug_utils.cc(14)] Hit debug scenario: 4
+ */
+const isDebugScenario4 = /Hit debug scenario: 4/
```

Comment:
I feel like someone could possibly write a test named this. Probably unlikely.....but possible. 😅

Reply:
We could add the file name to the regex (`debug_utils.cc`) or something else to make it more specific.

Reply:
Updated here: [`b4a3f38` (#30394)](https://github.com/cypress-io/cypress/pull/30394/commits/b4a3f38895052f69b3366890437b885ed7cdd215)